bin/commands/components/install/.exclusive-parameters

```diff
@@ -1,2 +1,3 @@
 component-file,component|o|string|required||||Path to the component package or directory.
```

Comment:
We should add notes in `bin/commands/components/install/.help` and explicitly say we will modify your YAML configuration if you don't pass `--skip-enable`. Something like this:

```
**IMPORTANT NOTES**, this command will by default enable the component globally by modifying your YAML configuration. You can pass `--skip-enable` to disable the behavior.
```

This is about considering zowe.yaml as user content, we don't want to modify it secretly.